wagtail/tests/manage.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+import os
+import sys
+
+
+if __name__ == "__main__":
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "wagtail.tests.settings")
+
+    from django.core.management import execute_from_command_line
+
+    execute_from_command_line(sys.argv)
```

Comment:
@thibaudcolas Is this needed for the linting to work? Personally, I think being able to run/interact with the test app locally is a really useful change, but unsure whether it's in scope for this PR.

Reply:
Oh boy 😄 this has nothing to do in here, thanks for catching this.